cardano-cli/test/Test/Golden/Shelley.hs

```diff
@@ -166,12 +159,6 @@ multiSigTests =
         [ ("golden_shelleyAllMultiSigAddressBuild", golden_shelleyAllMultiSigAddressBuild)
         , ("golden_shelleyAnyMultiSigAddressBuild", golden_shelleyAnyMultiSigAddressBuild)
         , ("golden_shelleyAtLeastMultiSigAddressBuild", golden_shelleyAtLeastMultiSigAddressBuild)
-        , ("golden_shelleyTransactionAssembleWitness_AllMultiSig", golden_shelleyTransactionAssembleWitness_AllMultiSig)
-        , ("golden_shelleyTransactionAssembleWitness_AnyMultiSig", golden_shelleyTransactionAssembleWitness_AnyMultiSig)
-        , ("golden_shelleyTransactionAssembleWitness_AtLeastMultiSig", golden_shelleyTransactionAssembleWitness_AtLeastMultiSig)
         , ("golden_shelleyTransactionAssembleWitness_SigningKey", golden_shelleyTransactionAssembleWitness_SigningKey)
-        , ("golden_shelleyTransactionAllMultiSigWitness", golden_shelleyTransactionAllMultiSigWitness)
-        , ("golden_shelleyTransactionAnyMultiSigWitness", golden_shelleyTransactionAnyMultiSigWitness)
-        , ("golden_shelleyTransactionAtLeastMultiSigWitness", golden_shelleyTransactionAtLeastMultiSigWitness)
```

Comment:
So we're removing a bunch of golden tests for the old way of providing scripts. That's fine. Are we going to get equivalent new ones for scripts within the tx body?

Reply:
Yep, I just wanted to confirm the changes to the cli parsers were ok before I wrote a bunch of tests.